inference-engine/tests/functional/plugin/shared/include/single_layer_tests/eltwise.hpp

```diff
@@ -7,44 +7,53 @@
 #include <gtest/gtest.h>
 
 #include <map>
+#include <functional_test_utils/layer_test_utils.hpp>
 
 #include "common_test_utils/common_layers_params.hpp"
 #include "common_test_utils/common_utils.hpp"
 #include "common_test_utils/test_common.hpp"
 #include "common_test_utils/test_constants.hpp"
 #include "ie_core.hpp"
 
-namespace EltwiseTestNamespace {
-
-    using ParameterInputIdx = int;
-    enum class InputLayerType {
-        CONSTANT,
-        PARAMETER
-    };
-    enum class EltwiseOpType {
-        ADD,
-        SUBSTRACT,
-        MULTIPLY
-    };
-    const char* InputLayerType_to_string(InputLayerType lt);
-    const char* EltwiseOpType_to_string(EltwiseOpType eOp);
-}// namespace EltwiseTestNamespace
+namespace LayerTestsDefinitions {
+namespace EltwiseParams {
+enum class InputLayerType {
+    CONSTANT,
+    PARAMETER
+};
+
+enum class OpType {
+    SCALAR,
+    VECTOR
+};
+
+enum class EltwiseOpType {
+    ADD,
+    SUBSTRACT,
```

Comment:
SUBTRACT

Reply:
Why it isn't fixed, but discussion resolved?